### DISCUSSION TITLE
Taking photos

### DISCUSSION THREAD

**Discussion by williambuchanan2** [dotnet/maui#14115] [Q&A]
### Description

I'm trying to do something which I thought would be very simple - take a photo. Cameras have been in phones for over a decade - but using Maui it seems that it is almost impossible to reliably take a photo, and it sounds like there is no clear path forward to fix the problem (please correct me if I have that wrong).

Issues I have at the moment:
1. Photos come out 90 degrees or upside down. And this is inconsistent on different devices so you never know how it's going to come out. I don't care what technical reasons are behind this - I would expect in the vast majority of cases most people expect the top of the photo to always be at the top.
2. Camera doesn't work on later versions of Android. Looks like a permission problem.
3. Lack of clarity on the way forward. Do I need to use some 3rd party tool to take photos or are the Maui team going to make something that works? The last I heard is that the Media Picker is a mess and going to be obsoleted - so where does that leave us?

What do I need to do to take a photo?

There must be some kind of recognition of how big a problem this is. Googling turns up a stream of people asking the same question since even the Xamarin days. There is some advice about using Exif - whatever that is - but no real instructions on how we are supposed to do this in a way that fixes the problem on all platforms.

A bit of a rant, and feel free to tell me that none of the above is a real problem, but it's getting really frustrating trying to work around all these problems. 

### Steps to Reproduce

Well documented already

### Link to public reproduction project repository

?

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, Windows

### Affected platform versions

All

### Did you find any workaround?

No

**Comment by markhodg1972** [comment]
This library DOES work, even though it never made it to final release:

https://github.com/jamesmontemagno/MediaPlugin

I would advocate using the standard library that is in .Net MAUI for managing the camera, however this library allows you to compress the image into a more useable size.
